python/orca/src/bigdl/orca/learn/pytorch/pytorch_metrics.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 import torch
+import torchmetrics
```

Comment:
This will introduce `torchmetrics` as orca's dependency, you may need to modify `setup.py`'s `install_requires`